core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorBodyDirective.js

```diff
@@ -28,10 +28,27 @@ oppia.directive('simpleEditorBody', [
         '$scope', 'SimpleEditorManagerService',
         'explorationSaveService', 'explorationRightsService',
         'explorationWarningsService', 'QuestionIdService',
-        function($scope, SimpleEditorManagerService,
+        'StatesToQuestionsService', function($scope, SimpleEditorManagerService,
             explorationSaveService, explorationRightsService,
-            explorationWarningsService, QuestionIdService) {
+            explorationWarningsService, QuestionIdService,
+            StatesToQuestionsService) {
           $scope.data = SimpleEditorManagerService.getData();
+          $scope.chooseQuestionType = false;
+          $scope.modalWrapperVisible = false;
```

Comment:
@seanlip It's used to close the interaction component when user clicks somewhere else, [See this video](https://streamable.com/vl3d8). Any suggestion on names?